pkg/controller/pxc/controller.go

```diff
@@ -1095,3 +1058,77 @@ func createOrUpdateConfigmap(cl client.Client, configMap *corev1.ConfigMap) erro
 
 	return nil
 }
+
+func (r *ReconcilePerconaXtraDBCluster) createOrUpdate(obj runtime.Object) error {
+	metaAccessor, ok := obj.(metav1.ObjectMetaAccessor)
+	if !ok {
+		return errors.New("Can't convert object to ObjectMetaAccessor")
+	}
+
+	objectMeta := metaAccessor.GetObjectMeta()
+
+	if objectMeta.GetAnnotations() == nil {
+		objectMeta.SetAnnotations(make(map[string]string))
+	}
+
+	delete(objectMeta.GetAnnotations(), "percona.com/last_config_hash")
+
+	hash, err := getObjectHash(obj)
```

Comment:
Please move this closer to place where it's used.

Reply:
054a623